### PR TITLE
SS-146 Public models page

### PR DESCRIPTION
**Pull request by alfredeen**
## Status

- [x] Ready
- [ ] Draft
- [ ] Hold

## Description

Reworked the public models view.

Jira [link](https://scilifelab.atlassian.net/browse/SS-146)

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [x] New feature
- [ ] Documentation update

## Checklist

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [x] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [ ] I have updated the release notes (releasenotes.md)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

### CODE REVIEW

File: templates/portal/apps.html [hunk 1]
```diff
@@ -3,124 +3,130 @@
...
+                                {% endif %}
                             </div>
-
-                            <div class="clearfix"></div>
                         </div>
 
-                        <div class="card-footer d-flex text-muted card-footer-text">
-                            <div class="col-3 d-inline-block">
-                                <span>{{ app.app.name }}</span>
+                        <div class="row mt-auto">
```
Comment (sandstromviktor) [inline templates/portal/apps.html:101]:
```suggestion
                        <div class="row mt-auto align-items-center">
```

File: templates/portal/apps.html [hunk 2]
```diff
@@ -3,124 +3,130 @@
...
+                            <div class="col-6">
+                                <a href="{{ app.table_field.url }}" target="_blank" class="btn btn-primary btn-sm">Open</a>
```
Comment (sandstromviktor) [inline templates/portal/apps.html:103]:
```suggestion
                                <a href="{{ app.table_field.url }}" target="_blank" class="btn btn-primary">Open</a>
```

**Review by sandstromviktor — APPROVED** [review]
Great job!

File: templates/portal/apps.html (hunk 2 above)
Comment (alfredeen) [reply templates/portal/apps.html:103]: Great suggestions. I will commit these as one change.

**Pull request merged**